liip/monitor-bundle/2.6/config/packages/monitor.yaml

```diff
@@ -0,0 +1,3 @@
+liip_monitor:
+    # enabling the controller requires that `assets` are enabled in the framework bundle
```

Comment:
Would it make sensio to auto-enable the controller based on the presence of assets (in a compiler pass in the bundle)?

Reply:
no .. I think most people will actually never enable the controller and only use this Bundle via the CLI